### PR TITLE
Added new recentComments, recentReplies

### PR DESCRIPTION
**Pull request by wyattjoh**
## What does this PR do?

- Adds new `recentComments` and `recentReplies` edges which are optimized for first loads

## How do I test this PR?

You can try the following query in the graphiql interface:

```graphql
{
  asset(url: "http://localhost:3000/") {
    id
    recentComments {
      id
      body
      recentReplies {
        id
        body
      }
    }
  }
}
```

### CODE REVIEW

File: graph/loaders/comments.js [hunk 1]
```diff
@@ -135,6 +136,95 @@ const getCommentsByQuery = ({user}, {ids, statuses, asset_id, parent_id, author_
...
+    {$match: {
+      parent_id: {
+        $in: ids
+      }
+    }},
+
+    // sort these by their created at timestamp, CHRONOLOGICAL'y as these are
+    // replies
+    {$sort: {
+      created_at: 1
```
Comment (riley) [inline graph/loaders/comments.js:152]: if we were getting the most recent, I think it's descending (`-1`)?

**Review by riley — CHANGES_REQUESTED** [review]

**Comment by riley** [comment]
I'm getting this error when I run your query in GraphiQL:

I have one approved comment in my db.

```
{
  "data": {
    "asset": {
      "id": "208eb834-a2e7-44cd-a614-ab8061e5fcf8",
      "recentComments": null
    }
  },
  "errors": [
    {
      "message": "exception: invalid operator '$slice'",
      "locations": [
        {
          "line": 4,
          "column": 5
        }
      ],
      "path": [
        "asset",
        "recentComments"
      ]
    }
  ]
}
```

File: graph/loaders/comments.js (hunk 1 above)
Comment (wyattjoh) [reply graph/loaders/comments.js:152]: This is for replies, which is CHRONOLOGICAL, or ascending, hence the `1`.

Comment (riley) [reply graph/loaders/comments.js:152]: ah right. nice.

**Comment by riley** [comment]
the `$slice` failure was operator error: my mongodb version was old.

**Review by riley — APPROVED** [review]

**Pull request merged**